### ISSUE TITLE
"cachejre" not writing JRE to correct location?

### ISSUE THREAD

**Issue by zeobviouslyfakeacc** [libgdx/packr#143]
Hi everyone!

I've been trying to use the "cachejre" command line / JSON option, but I can't get Packr to copy the JRE to the "cachejre" path I've specified.

I'm really not sure if I'm just doing something wrong or if this is a bug in Packr, but I found [this line of code](https://github.com/libgdx/packr/blob/4dd00a1fb5075dc1b6cd84f46bfdd918eb3d50e9/src/main/java/com/badlogicgames/packr/Packr.java#L285) in Packr.java that doesn't make sense to me:
```java
		if (extractToCache) {
			// if cache is used, copy again here; if the JRE is cached already,
			// this is the only copy done (and everything above is skipped)
-->			PackrFileUtils.copyDirectory(jreStoragePath, output.resourcesFolder);
		}
```

Shouldn't the JRE be copied to the `config.cacheJre` path that is checked earlier in the same method?

**Comment by code-disaster** [comment]
jreStoragePath is initialized to config.cacheJre [here](https://github.com/libgdx/packr/blob/4dd00a1fb5075dc1b6cd84f46bfdd918eb3d50e9/src/main/java/com/badlogicgames/packr/Packr.java#L232) in this case.

I don't see a reason why it shouldn't work, though I'm not using the command line or config JSON myself. These options follow the same pattern as config.outDir, so they take a path relative to the working directory.

**Comment by zeobviouslyfakeacc** [comment]
Ah, it's indeed user error: I used the wrong artifact on maven.

I expected packr to be on maven central, so I just searched for "packr" and didn't verify the group id, so I ended up with `com.bladecoder.packr:packr:2.1` instead of `com.badlogicgames.packr:packr:2.1-SNAPSHOT` from your snapshots repo.

With the correct repo added and the group id corrected, everything is now working as expected. Thanks for your help! 😄
